packages/new-polymath-ui/src/components/TooltipIcon/styles.ts

```diff
@@ -30,5 +30,5 @@ export const Tooltip = styled(TooltipComponent)<Props>`
   border: 1px solid #dfe3e6;
   word-wrap: break-word;
   font-weight: normal;
-  font-size: ${({ theme }) => theme.fontSizes.baseText};
+  font-size: ${({ theme }) => theme.fontSizes[2]};
```

Comment:
I think the font size was right already, if you check the Invision. From what I see was reported by Stan, it's the `line-height` that is wrong.

Reply:
Thanks. Resolved.

Reply:
Sorry to be annoying again, but could you use the value from the theme for that instead of hardcoded value? 😅
I believe `theme.lineHeights.normal` should work fine!
As a general rule, try to hardcode only if it's not in theme (or if the theme doesn't fit!).

Reply:
No problem